connection/connection.ts

```diff
@@ -679,8 +683,7 @@ export class Connection {
 
     const buffer = this.#packetWriter.addCString(query.text).flush(0x51);
 
-    await this.#bufWriter.write(buffer);
-    await this.#bufWriter.flush();
+    await this.#connWritable.write(buffer);
```

Comment:
Why are we no longer flushing the buffer?

Reply:
We aren't flushing the writer anymore since the original code was flushing the data from the BufWriter class (from @std/io) to the underlying Deno.Writer, but since we're not using either of those and are using normal web streams here, awaiting the write seems to be fine in making sure everything gets written